spaceship/lib/spaceship/portal/provisioning_profile.rb

```diff
@@ -457,7 +457,7 @@ def update!
             devices.map(&:id),
             mac: mac?,
             sub_platform: tvos? ? 'tvOS' : nil,
-            template_name: template_name
+            template_name: is_template_profile ? template.purpose_name : nil
```

Comment:
Did I understand correctly that this additional `is_template_profile` conditional is to make this equal to the previous `template_name`, which is actually a call to a method, that includes this conditional?
Can't we remove that method then - or change it to using the correct field instead of replacing the call here? The other `template_name` usages in this file seem to be for a method parameter, and not that method.

(Don't know this piece of code very well)

Reply:
Seems fair. Initially I thought this function was being used elsewhere but I'll drop the method